### PR TITLE
HParams: Do not scroll regex filter in the horizontal direction

### PR DESCRIPTION
**Pull request by JamesHollyer**
## Motivation for features / changes
I originally started to fix a bug where the bottom border in the regex did not stretch to the end of the table once scrolled. However, I decided there is no reason to scroll the regex at all and it can actually be kind of annoying. So I change the x scroll to just be the table

## Screenshots of UI changes 

(or N/A)

### CODE REVIEW

File: tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
 limitations under the License.
 -->
 
+<!-- <div clas="runs-data-table-container"> -->
```
Comment (rileyajones) [inline tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html:18]:
```suggestion
```

File: tensorboard/webapp/runs/views/runs_table/runs_data_table.scss [hunk 2]
```diff
@@ -115,12 +115,16 @@ tb-data-table-header-cell:last-of-type {
...
-  padding: 0 16px;
+  width: 100%;
 
   tb-filter-input {
+    padding-left: 16px;
     flex-grow: 1;
   }
 }
+.table-container {
+  overflow-x: scroll;
```
Comment (rileyajones) [inline tensorboard/webapp/runs/views/runs_table/runs_data_table.scss:126]:
```suggestion
  overflow-x: auto;
```

File: tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html (hunk 1 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html]: Oops. Removed.

File: tensorboard/webapp/runs/views/runs_table/runs_data_table.scss (hunk 2 above)
Comment (JamesHollyer) [reply tensorboard/webapp/runs/views/runs_table/runs_data_table.scss]: Yea, no reason. Changed.

**Pull request merged**